src/views/Profile/components/PersonalEdit.vue

```diff
@@ -0,0 +1,132 @@
+    lastname: validateField(localForm.value.lastname, wordValidation),
+    phoneNumber: validateField(localForm.value.phoneNumber, phoneNumberValidation)
+  }
+})
+</script>
+
+<template>
+  <div class="accordion-body">
+    <div class="mb-3">
+      <label for="email" class="form-label">Correo Electrónico</label>
+      <input
+        v-model="localForm.email"
+        :class="{
+          'is-invalid': localForm.email && validationErrors.email.length > 0
+        }"
+        type="email"
+        id="email"
+        placeholder="Ingresa tu correo"
+        class="form-control"
+      />
+      <span v-if="localForm.email && validationErrors.email.length > 0" class="text-danger">
+        {{ validationErrors.email[0] }}
+      </span>
+    </div>
+    <div class="mb-3">
+      <label for="name" class="form-label">Nombre</label>
+      <input
+        v-model="localForm.name"
+        :class="{
+          'is-invalid': localForm.name && validationErrors.name.length > 0
+        }"
+        type="text"
+        id="name"
+        placeholder="Ingresa tu nombre"
+        class="form-control"
+      />
+      <span v-if="localForm.name && validationErrors.name.length > 0" class="text-danger">
+        {{ validationErrors.name[0] }}
+      </span>
+    </div>
+    <div class="mb-3">
+      <label for="lastname" class="form-label">Apellido</label>
+      <input
+        v-model="localForm.lastname"
+        :class="{
+          'is-invalid': localForm.lastname && validationErrors.lastname.length > 0
+        }"
+        type="text"
+        id="lastname"
+        class="form-control"
+        placeholder="Ingresa tu apellido"
+      />
+      <span v-if="localForm.lastname && validationErrors.lastname.length > 0" class="text-danger">
+        {{ validationErrors.lastname[0] }}
+      </span>
+    </div>
+    <div class="mb-3">
+      <label for="phoneNumber" class="form-label">Número de Teléfono</label>
+      <div class="input-group">
+        <select class="form-select w-25" v-model="localForm.countryCode">
+          <option v-for="code in countryCodes" :key="code.country" :value="code.callingCode">
+            {{ code.country }} (+{{ code.callingCode }})
+          </option>
+        </select>
+        <input
+          v-model="localForm.phoneNumber"
+          :class="{
+            'is-invalid': localForm.phoneNumber && validationErrors.phoneNumber.length > 0
+          }"
+          type="text"
+          id="phoneNumber"
+          class="form-control w-75"
+          placeholder="Ingresa tu número telefónico"
+        />
+      </div>
+      <span
+        v-if="localForm.phoneNumber && validationErrors.phoneNumber.length > 0"
+        class="text-danger"
+      >
+        {{ validationErrors.phoneNumber[0] }}
+      </span>
+    </div>
+    <div class="mb-3">
+      <label for="profilePictureUrl" class="form-label">Subir una foto de perfil</label>
+      <BaseFileUpload @file-selected="onFileSelected" />
```

Comment:
This is great but this is an input, we should try to add the component accoding to the directory describes better their behavior, by the way for other component that are share along the app, we can create a shared directory, for example the loader can be there